src/components/game/index.jsx

```diff
@@ -77,15 +78,19 @@ class GameScreen extends React.Component {
     this.refs.controls.update();
     this.refs.stats.forceUpdate();
 
-    const cameraRotated = this.prevCameraRotation === null ||
-      !this.prevCameraRotation.equals(this.camera.quaternion);
+    const cameraMoved =
+      this.prevCameraRotation === null ||
+      this.prevCameraPosition === null ||
+      !this.prevCameraRotation.equals(this.camera.quaternion) ||
+      !this.prevCameraPosition.equals(this.camera.position);
 
-    session.world.animate(this.clock.getDelta(), this.camera, cameraRotated);
+    session.world.animate(this.clock.getDelta(), this.camera, cameraMoved);
```

Comment:
I guess I viewed the name as reflecting translation + rotation (both are forms of movement). I suppose, in theory, something like the FOV changing could also impact billboarding, so maybe changed would make more sense.

Okay if I do that as a follow up commit? I'm dependent on this commit in another PR I'm about to make.